docs/_data/portland-2022-schedule.yaml

```diff
@@ -82,17 +82,19 @@ talks_day2:
   - duration: '0:05'
     title: '5 minute break'
   - duration: '0:15'
-    title: 'Q&A for the two previous talks'
+    title: '✋ Q&A for the two previous talks'
   - duration: '0:15'
     title: 'Coffee break'
   - duration: '0:30'
     slug: don-t-trust-the-numbers-kumar-dhanagopal
   - duration: '0:05'
     title: '5 minute break'
-  - duration: '0:30'
-    slug: inclusive-language-victor-bracht
+  - duration: '0:25'
+    title: "<b>Meetup information session - Get involved locally!</b>"
+  - duration: '0:05'
```

Comment:
can we do 10 minutes so we can talk about Quorum?

Reply:
It's currently scheduled for 25 mins (The duration is above it)

Reply:
It's currently scheduled for 25 mins (The duration is above it)